pkg/configure/oneagent/pmc/pmc.go

```diff
@@ -38,5 +49,12 @@ func Configure(log logr.Logger, fs afero.Afero, inputDir, targetDir string) erro
 		return err
 	}
 
-	return UpdateInPlace(log, fs, targetDir, conf)
+	conf.InstallPath = &installPath
+
+	source := GetSourceRuxitAgentProcPath(targetDir)
+	destination := GetDestinationRuxitAgentProcPath(configDir)
```

Comment:
```suggestion
	dscFile := GetDstRuxitAgentProcFilePath(configDir)
```

Reply:
used `dscPath` as the `dstFile` is the actual open file